addressing_address_map.go

```diff
@@ -0,0 +1,202 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// AddressMap contains information about an address map.
+type AddressMap struct {
+	ID           string                 `json:"id"`
+	Description  string                 `json:"description"`
+	DefaultSNI   string                 `json:"default_sni"`
```

Comment:
```suggestion
	DefaultSNI   *string                 `json:"default_sni"`
```

Same here, this field can be null.